cypress/e2e/accessibility.cy.ts

```diff
@@ -27,7 +27,7 @@ describe("Accessibility tests in light mode", () => {
         cy.get("button[aria-label='Sign Out Button']").click();
         cy.url().should("include", "/login");
         // Forces Page to Render Fully
-        cy.contains("Sign in", { timeout: 3000 }).should("be.visible");
+        cy.contains("Sign in").should("be.visible");
```

Comment:
Is there a reason you've removed the timeout: 3000?

Reply:
Inbuilt 4000ms timeout